consumers/email-consumer/main.py

```
@@ -0,0 +1,60 @@
+topic = "email"
+print(f"Connecting to Kafka at {conf['bootstrap.servers']}")
+consumer = Consumer(conf)
+consumer.subscribe([topic])
+
+try:
+    while True:
+        msg = consumer.poll(timeout=1.0)
+        if msg is None:
+            continue
+        if msg.error():
+            print("Consumer error: {}".format(msg.error()))
```

Ruff (0.8.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)